magento/pwa-studio
# [PWA-811] Provide ability to translate Venia's header / footer

## Description

Provide the ability to store translation strings locally, pull correct locale from provided store view code, and translate the header and footer.

## Related Issue
https://jira.corp.magento.com/browse/PWA-811

## Acceptance 




### Verification Stakeholders
@jimbo 
@dpatil-magento 

### Specification



### Verification Steps


1. Create new store in Magento with different language (e.g. French)
2. Update env variable to match store view code
3. Add a json file with translation strings in the `i18n` directory
4. Ensure strings in the header and footer are translated

## Screenshots / Screen Captures (if appropriate)

## Checklist
